src/main/java/com/mercuryCyclists/Sales/controller/OnlineSaleController.java

```diff
@@ -0,0 +1,28 @@
+package com.mercuryCyclists.Sales.controller;
+
+import com.mercuryCyclists.Sales.entity.OnlineSale;
+import com.mercuryCyclists.Sales.service.OnlineSaleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@CrossOrigin
+@RestController
+@RequestMapping(path = "api/v1/online-sale")
+public class OnlineSaleController {
+
+    private final OnlineSaleService onlineSaleService;
+
+    @Autowired
+    public OnlineSaleController(OnlineSaleService onlineSaleService) {
+        this.onlineSaleService = onlineSaleService;
+    }
+
+    /**
+     * Register a new Online Sale
+     */
+    @PostMapping
```

Comment:
No get endpoints?

Reply:
or updates, etc? Im assuming your still doing it

Reply:
https://github.com/pedrooste/MercuryCyclistsSales/blob/c64b7a284d4f924acaabec246cebce3c092adb15/src/main/java/com/mercuryCyclists/Sales/controller/OnlineSaleController.java